src/Uno.UI.Wasm/ts/Clipboard.ts

```diff
@@ -17,5 +34,12 @@
 
 			return "ok";
 		}
+
+		private static onClipboardChanged() {
+			if (!Clipboard.dispatchContentChanged) {
+				Clipboard.dispatchContentChanged = (<any>Module).mono_bind_static_method("[Uno] Windows.ApplicationModel.DataTransfer.Clipboard:DispatchContentChanged");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5540286)